### ISSUE TITLE
[FEATURE] Choose render strategy

### ISSUE THREAD

**Issue by rabits** [state-of-the-art/BlendNet#90]
### Is your feature request related to a problem? Please describe.
Right now BlendNet render strategy is simple - to make sure the frame will be rendered as soon as possible on all the provided resources. That could be useful to render in parallel (one frame per instance or per number of instances).

reported by: @heizermeister

### Describe the solution you'd like
I think `render strategies` could be implemented to distribute the workloads in different ways. I think the user could choose the strategy in Addon UI during the Manager startup.

**Comment by rabits** [comment]
Probably that will be handy to have a checkbox near the Agent (in UI agents list) - which will say to use it or not during running the Image or Animation. Also need UI to select/deselect all of course.

**Comment by rabits** [comment]
There is a number of interesting questions regarding cloud rendering for new render strategies. For example - how to make sure it will save the frame intermediate results, how to split the resources if we're going to run a number of tasks on one agent...

**Comment by juangea** [comment]
I imagine this has not evolved too much yet, we are eagerly waiting for a frame-per-node or "couple-of-frames-per-node" render strategy for animation rendering :)

**Comment by rabits** [comment]
Yeah, sorry about that, just figured out the Azure provider #30 so will start to work on this one soon)

**Comment by juangea** [comment]
Awesome, thanks :)

**Comment by rabits** [comment]
Ok, I think the first idea is to have one frame per agent and to switch between those extremums by a list of options to add more when they will be discovered. For example the VSE rendering will require to have a number of sequential frames to render on the same agent at a time.

**Comment by rabits** [comment]
Oh, actually this one requires #65

**Comment by rabits** [comment]
Prepared a plan on rewriting the schedule engine to support the next render schemas:
![BlendNet render schemas](https://user-images.githubusercontent.com/1093323/113521581-3ccdb000-954f-11eb-83e1-ed104b083ecf.png)
